frontend/src/components/Metrics/IstioMetrics.tsx

```diff
@@ -401,7 +401,7 @@ class IstioMetricsComponent extends React.Component<Props, MetricsState> {
 
 const mapStateToProps = (state: KialiAppState) => {
   return {
-    jaegerIntegration: state.jaegerState.info ? state.jaegerState.info.integration : false,
+    jaegerIntegration: state.tracingState.info ? state.tracingState.info.integration : false,
```

Comment:
If the state property is generic (tracingState), the variable should be renamed to tracingIntegration